classes/TaskRunner/Miscellaneous/UpdateConfig.php

```diff
@@ -174,4 +182,57 @@ private function writeConfig(array $config): bool
 
         return (new UpgradeConfigurationStorage($this->container->getProperty(UpgradeContainer::WORKSPACE_PATH) . DIRECTORY_SEPARATOR))->save($this->container->getUpgradeConfiguration(), UpgradeFileNames::CONFIG_FILENAME);
     }
+
+    /**
+     * @throws ZipActionException
+     * @throws Exception
+     */
+    private function extractPrestashopVersionFromZip(string $zipFile): string
+    {
+        $internalZipFileName = 'prestashop.zip';
+        $versionFile = 'install/install_version.php';
+
+        if (!file_exists($zipFile)) {
+            throw new FileNotFoundException("Unable to find $zipFile file");
```

Comment:
In think these exceptions messages might be not visible in the logs. Is that expected?

Reply:
Well seen, you have to change the error log to replace it with the exception potentially raised